src/common/roles.js

```
@@ -0,0 +1,40 @@
+const { findWithId } = require("../mongo/mongo");
+const { lcns } = require("../utils/util");
+const { ROLES_ID } = require('../config');
+
+var roleNames = [];
+const rolesDb= getRolesDB();
+
+async function loadRoles() {
+    try {
+        return await findWithId(ROLES_ID);
+    } catch (e) {
+        console.log(`Could not load roles from database`);
+        console.log(e)
```

Are we happy to just silently fail? Could just throw the exception here too to cause a failure?

---

Wrapped the call in try/catch